From 5c8815ee988c5ec9fca11ef162451b9ddfb4be81 Mon Sep 17 00:00:00 2001
From: Nelson Liu <nelson.liu@mediatek.com>
Date: Thu, 8 Sep 2016 16:18:06 +0800
Subject: [PATCH 05/19] weston: do not check master fd for DRM driver

give access to other module to use drm.
Test: test ok

Change-Id: If7500f1c949545c71977b11b66c5767bcf2377b3
Signed-off-by: Nelson Liu <nelson.liu@mediatek.com>
CR-Id: AUTO00000252
---
 src/launcher-direct.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/launcher-direct.c b/src/launcher-direct.c
index 29d9c28..0fd099e 100644
--- a/src/launcher-direct.c
+++ b/src/launcher-direct.c
@@ -220,11 +220,6 @@ launcher_direct_open(struct weston_launcher *launcher_base, const char *path, in
 
 	if (major(s.st_rdev) == DRM_MAJOR) {
 		launcher->drm_fd = fd;
-		if (!is_drm_master(fd)) {
-			weston_log("drm fd not master\n");
-			close(fd);
-			return -1;
-		}
 	}
 
 	return fd;
-- 
1.9.1

